apps/browser/src/vault/popup/settings/archive.component.html

```diff
@@ -0,0 +1,82 @@
+<popup-page>
+  <popup-header slot="header" [pageTitle]="'archive' | i18n" showBackButton>
+    <ng-container slot="end">
+      <app-pop-out></app-pop-out>
+    </ng-container>
+  </popup-header>
+
+  @if (archivedCiphers$ | async; as archivedItems) {
+    @if (archivedItems.length) {
+      <bit-section>
+        <bit-section-header>
+          <h2 bitTypography="h6">
+            {{ "itemsInArchive" | i18n }}
+          </h2>
+          <span bitTypography="body1" slot="end">{{ archivedItems.length }}</span>
+        </bit-section-header>
+        <bit-item-group>
+          @for (cipher of archivedItems; track cipher.id) {
+            <bit-item>
+              <button
+                bit-item-content
+                type="button"
+                [appA11yTitle]="'viewItemTitle' | i18n: cipher.name"
+                (click)="view(cipher)"
+              >
+                <div slot="start" class="tw-justify-start tw-w-7 tw-flex">
+                  <app-vault-icon [cipher]="cipher"></app-vault-icon>
+                </div>
+                <span data-testid="item-name">{{ cipher.name }}</span>
+                @if (cipher.hasAttachments) {
+                  <i class="bwi bwi-paperclip bwi-sm" [appA11yTitle]="'attachments' | i18n"></i>
+                }
+                <span slot="secondary">{{ cipher.subTitle }}</span>
+              </button>
+              <bit-item-action slot="end">
+                <button
+                  type="button"
+                  bitIconButton="bwi-ellipsis-v"
+                  size="small"
+                  label="{{ 'options' | i18n }}"
+                  [attr.aria-label]="'moreOptionsLabel' | i18n: cipher.name"
+                  [title]="'moreOptionsTitle' | i18n: cipher.name"
```

Comment:
⛏️ `appA11yTitle` can be used over the two aria-label and title definitions

```suggestion
                  [appA11yTitle]="'moreOptionsTitle' | i18n: cipher.name"
```